climatechange.html

```diff
@@ -19,7 +19,8 @@
     <meta property="og:description"
         content="Charity Earth - Tell us what you value - The fun way to discover new environmental charities!
     Take a short quiz and based on your areas of interest we will suggest a charity that would love a donation from you." />
-    <meta name="image" property="og:image" content="assets/images/charity-earth-responsive.png" />
```

Comment:
Can you pop this line back in on the pages - it's the Link to image for social sharing